modules/HandlerOfInputsFromUi.py

```diff
@@ -19,61 +22,16 @@ def __init__(self):
             for category in self.mod_get.get_categories_details(): self.categories[category["id"]]=category["name"]
         except error.IsOrieteeringApiError as e:
             raise e
-        self.cache = {
-            "competitions": {
-                1888: """
-                {
-                    "id": "1888",
-                    "title_sk": "STRED O LIGA  - 2. kolo",
-                    "date_from": "2024-12-04",
-                    "date_to": "2024-12-04",
-                    "cancelled": "0",
-                    "deadline": "2024-12-01",
-                    "events": {
-                        "id": "688"
-                    },
-                    "categories": [
-                        {
-                            "id": "97531",
-                            "category_id": "160",
-                            "category_name": "A - muži"
-                        },
-                        {
-                            "id": "97541",
-                            "category_id": "161",
-                            "category_name": "A - ženy"
-                        },
-                        {
-                            "id": "97551",
-                            "category_id": "162",
-                            "category_name": "B - muži"
-                        },
-                        {
-                            "id": "98321",
-                            "category_id": "163",
-                            "category_name": "B - ženy"
-                        },
-                        {
-                            "id": "98331",
-                            "category_id": "164",
-                            "category_name": "C - muži"
-                        },
-                        {
-                            "id": "98341",
-                            "category_id": "165",
-                            "category_name": "C - ženy"
-                        }
-                    ]
-                }
-                """
-            }
-        }
-        # for category in self.mod_get.get_categories_details():
-        #     self.categories[category["id"]] = category["name"]
+        
+        self.races = {} # dict filled with dicts of races, see function fil_out_cache 
+        self.club_id = 81  # ulozene v configu
```

Comment:
toto uz bude v configu, ak spustime config_file_reader.py, tak ono to vygeneruje, to by sa malo potom dopisat do instalacky

Reply:
jop, asi som nedal uplne dobry koment ale tak to bolo myslene